### ISSUE TITLE
Identify first 10 test cases for pilot

### ISSUE THREAD

**Issue by drewnielson** [Section508Coordinators/baselinealignment#151]

**Comment by kengdoj** [comment]
3/3/22:

Kathy's suggestions:
1.	Image - no accessible name or description
2.	Meaningful Image - with non-descriptive alt text
3.	Contrast - insufficient contrast ratio for small text 
4.	Visual Headings - most important heading does not have the higher priority level programmatically 
5.	Visually apparent unordered list not identified as a list programmatically 
6.	Language of Page - wrong language identified
7.	Language of Parts - not identified 
8.	Visual Headings - no visual headings
9.	Form Names - accessible name from aria-label attribute
10.	Changing Content - aria-live='off'

Andrew likes, and add:
•	Data table - no <th> elements (TC12.1-4-fail-1)
